subsys/bluetooth/host_extensions/host_extensions.c

```diff
@@ -72,3 +74,25 @@ int bt_conn_set_power_control_request_params(struct bt_conn_set_pcr_params *para
 	return bt_hci_cmd_send_sync(BT_HCI_OP_SET_POWER_CONTROL_REQUEST_PARAMS, buf, NULL);
 }
 #endif /* CONFIG_BT_TRANSMIT_POWER_CONTROL */
+
+#if defined(CONFIG_BT_CTLR_ADV_EXT)
+int bt_host_extension_reduce_initator_aux_channel_priority(bool reduce)
+{
+	struct bt_hci_set_role_priority *cmd;
+	struct net_buf *buf;
+
+	buf = bt_hci_cmd_create(BT_HCI_OP_SET_ROLE_PRIORITY, sizeof(*cmd));
```

Comment:
Usually we use `BT_HCI_OP_VS_xx`. I see the one above slipped, but it's good if future extensions are namespaced.

Reply:
should I update the above?

Reply:
it's best in another PR that just does the renaming. I think @alwa-nordic talked about some GCC builtin to do a non-breaking migration/deprecation.